dart/lib/src/protocol/breadcrumb.dart

```diff
@@ -165,28 +170,37 @@ class Breadcrumb {
     if (data != null) {
       data = Map<String, dynamic>.from(data as Map);
     }
-
     return Breadcrumb(
       timestamp: timestamp != null ? DateTime.tryParse(timestamp) : null,
       message: json['message'],
       category: json['category'],
       data: data,
       level: levelName != null ? SentryLevel.fromName(levelName) : null,
       type: json['type'],
+      unknown: unknownFrom(json, {
+        'level',
+        'timestamp',
+        'data',
+        'message',
+        'category',
+        'type',
+      }),
     );
   }
 
   /// Converts this breadcrumb to a map that can be serialized to JSON according
   /// to the Sentry protocol.
   Map<String, dynamic> toJson() {
-    return {
+    final json = {
       'timestamp': formatDateAsIso8601WithMillisPrecision(timestamp),
       if (message != null) 'message': message,
       if (category != null) 'category': category,
       if (data?.isNotEmpty ?? false) 'data': data,
       if (level != null) 'level': level!.name,
       if (type != null) 'type': type,
     };
+    json.addAll(unknown ?? {});
+    return json;
```

Comment:
This can be simplified with the spread operator:
```suggestion
    return {
      'timestamp': formatDateAsIso8601WithMillisPrecision(timestamp),
      if (message != null) 'message': message,
      if (category != null) 'category': category,
      if (data?.isNotEmpty ?? false) 'data': data,
      if (level != null) 'level': level!.name,
      if (type != null) 'type': type,
      ...?unknown
    };
```